### ISSUE TITLE
Style/IfUnlessModifier funky autocorrect when if/unless modifier already exists

### ISSUE THREAD

**Issue by jbodah** [bbatsov/rubocop#2553]
Autocorrect produces a bit of a weird result for `IfUnlessModifier` when an `if/unless` is already being used in the wrapped expression

```rb
if opts[:yield_in_batches]
  yield(batch) if batch.present?
end
```

```sh
$ rubocop --only IfUnlessModifier check.rb --auto-correct
$ cat check.rb
yield(batch) if batch.present? if opts[:yield_in_batches]
```

Seems like it compiles properly, but it would be nicer to combine the expressions if possible

**Comment by jbodah** [comment]
Looks like this is handled by https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/nested_modifier.rb so closing

**Comment by jonas054** [comment]
RuboCop 0.35.1 corrects the code to
```ruby
yield(batch) if opts[:yield_in_batches] && batch.present?
```
Seems you're running an older version.

**Comment by jbodah** [comment]
@jonas054 might be something with the parser version (I'm running 2.1.3 locally)

**Comment by jbodah** [comment]
Oh I think it might be that I disabled https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/nested_modifier.rb so that isn't fixing things again after the first autocorrect

**Comment by jonas054** [comment]
This conversation looks a bit strange now, because I deleted my comment (that @jbodah probably was running an older version) when I saw that he closed the issue himself.

Good that you resolved it, anyway!
